pmdarima/arima/tests/test_auto.py

```diff
@@ -17,3 +17,36 @@ def test_deprecation_warnings_on_class():
     with pytest.warns(DeprecationWarning) as we:
         auto.AutoARIMA(sarimax_kwargs={"simple_differencing": True})
     assert we
+
+
+def test_issue_341():
+    y = [0, 132, 163, 238, 29, 0, 150, 320, 249, 224, 197, 31, 0, 154,
+         143, 132, 135, 158, 21, 0, 126, 100, 137, 105, 104, 8, 0, 165,
+         191, 234, 253, 155, 25, 0, 228, 234, 265, 205, 191, 19, 0, 188,
+         156, 172, 173, 166, 28, 0, 209, 160, 159, 129, 124, 18, 0, 155]
+
+    try:
+        auto.auto_arima(
+            y,
+            start_p=1,
+            start_q=1,
+            test='adf',
+            max_p=3,
+            max_q=3,
+            m=52,
+            start_P=0,
+            seasonal=True,
+            d=None,
+            D=1,
+            trace=True,
+            error_action='ignore',
+            suppress_warnings=True,
+            stepwise=True
+        )
+
+    # assert that we catch the np LinAlg error and reraise with a more
+    # meaningful message
+    except ValueError as v:
```

Comment:
I think the [preferred way](https://docs.pytest.org/en/stable/assert.html#assertions-about-expected-exceptions) to do this in pytest is like this:

```python
with pytest.raises(ValueError) as error:
    auto.auto_arima(
        y,
        start_p=1,
        start_q=1,
        test='adf',
        max_p=3,
        max_q=3,
        m=52,
        start_P=0,
        seasonal=True,
        d=None,
        D=1,
        trace=True,
        error_action='ignore',
        suppress_warnings=True,
        stepwise=True
    )

assert "Encountered exception in stationarity test" in str(error.value)
```

Reply:
Or I guess you could use `pmdarima.compat.pytest.pytest_error_str` as we do in `test_utils.py`

Reply:
Yes, we should. Not sure why I didn't do that here.